src/DurableTask/TaskOrchestrationContext.cs

```diff
@@ -24,6 +25,11 @@ public abstract class TaskOrchestrationContext
     /// </summary>
     public abstract string InstanceId { get; }
 
+    /// <summary>
+    /// Gets the parent instance. Null if no parent orchestration.
```

Comment:
nit, this style is a bit more consistent with other properties: `"Gets the parent instance or <c>null</c> if there is no parent orchestration."`